exonum/src/storage/proof_map_index/key.rs

```diff
@@ -462,6 +468,53 @@ impl PartialOrd for ProofPath {
     }
 }
 
+impl ProofPath {
+    /// Creates a compressed binary representation of the given proof path.
+    ///
+    /// # Binary format
+    ///
+    /// - **bits_len** - total length in bits compressed by the leb128 algorithm.
+    /// - **bytes** - non-null bytes of the given ProofPath, i.e. the first `(bits_len + 7) / 8` bytes.
+    pub fn compress(&self) -> SmallVec<[u8; 64]> {
```

Comment:
Should it be the part of the public (user-available) API? Are there any scenarios when the user would need it?

Reply:
This needs only for exonum clients.